documentation/api/logs.md

```diff
@@ -34,13 +40,14 @@ The default host address for these routes is `https://localhost:52323`. This rou
 |---|---|---|---|---|
 | `pid` | path | false | int | The ID of the process. |
 | `uid` | path | false | guid | A value that uniquely identifies a runtime instance within a process. |
+| `name` | path | false | string | (Preview 5+) The name of the process. |
```

Comment:
nit: I understand we want to call out Preview 5+ now in documentation now since the feature hasn't shipped yet, but I feel like we should remove these at some point.

Reply:
Omit the preview5+ in here. Let's version the docs with the code.

Perhaps, we have a separate PR to move the current docs to the release/5.0 branch and then modify the docs as we see fit?

Reply:
If we do that, then we need to have some better differentiation (maybe with different links) where released bits point to the docs in the release branch rather than the main branch.

Reply:
We could update the fwlinks to point to docs in the most recent release branch?

Reply:
The other thing I've found that works well with mardown doc is add a yaml header with metadata like what version the docs address

Reply:
https://github.com/dotnet/docs/blob/main/docs/core/diagnostics/index.md